src/python/terminal.py

```diff
@@ -228,6 +233,21 @@ def _launch_nested_container_session(self):
                     'value': self.cmd,
                     'arguments': [self.cmd] + self.args,
                     'shell': False}}}
+        if self.env is not None:
+            env_vars = []
+            env_var_regex = re.compile('(.*)=(.*)$')
```

Comment:
I would have done a simple `.split('=')` because it's simpler but if you want to go with regexp I think using fullmatch (https://docs.python.org/3/library/re.html#re.Pattern.fullmatch) with '([A-Z0-9]+)=(.*)' would be better.

Reply:
You're right I used regex because it allowed non-escaped `=` on the value side for free, but I didn't have the proper regex. From [what I understand](https://stackoverflow.com/a/2821183/460005), such a regex would be:
```
^([A-Z_][A-Z0-9_]*)=(.*)$
```
In action: https://regex101.com/r/cTewrl/1
I'll update the code.